src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderNonAssociation.kt

```diff
@@ -34,13 +33,11 @@ data class OffenderNonAssociation(
   @EmbeddedId
   val id: OffenderNonAssociationId,
 
-  @ManyToOne(optional = false, fetch = FetchType.LAZY)
-  @JoinColumn(name = "OFFENDER_BOOK_ID", nullable = false)
-  val offenderBooking: OffenderBooking,
+  @Column(name = "OFFENDER_BOOK_ID", nullable = false)
+  val offenderBookingId: Long,
 
-  @ManyToOne(optional = false, fetch = FetchType.LAZY)
-  @JoinColumn(name = "NS_OFFENDER_BOOK_ID", nullable = false)
-  val nsOffenderBooking: OffenderBooking,
+  @Column(name = "NS_OFFENDER_BOOK_ID", nullable = false)
+  val nsOffenderBookingId: Long,
```

Comment:
Moving to using ids avoids a lot of unnecessary offender and booking data from being fetched when getting NAs